tools/interop_matrix/patches/csharp_v1.0.1/git_repo.patch

```diff
@@ -5,13 +5,6 @@ index c880e42ba1..70ef9596bb 160000
 @@ -1 +1 @@
 -Subproject commit c880e42ba1c8032d4cdde2aba0541d8a9d9fa2e9
 +Subproject commit 70ef9596bbcc11353b9bb8d4e91478694dd21439
-diff --git a/third_party/gflags b/third_party/gflags
-index 05b155ff59..30dbc81fb5 160000
---- a/third_party/gflags
-+++ b/third_party/gflags
-@@ -1 +1 @@
--Subproject commit 05b155ff59114735ec8cd089f669c4c3d8f59029
-+Subproject commit 30dbc81fb5ffdc98ea9b14b1918bfe4e8779b26e
```

Comment:
This file shouldn't be modified because it is used on the old branch which has to use `gflag`.

Reply:
While this is no longer part of this PR, I have reverted it from that commit.